src/base/flamec/supermatrix/hip/main/FLASH_Queue_hip.c

```diff
@@ -236,7 +236,6 @@ dim_t FLASH_Queue_get_hip_num_blocks( void )
    return flash_queue_hip_n_blocks;
 }
 
-
```

Comment:
Will revert this.

Reply:
Fixed: https://github.com/iotamudelta/libflame/pull/2/commits/bd8b490c69720ca6210141fe13ac0703afc20b18